v21.2/recommended-production-settings.md

```diff
@@ -21,66 +21,90 @@ We recommend running a [glibc](https://www.gnu.org/software/libc/)-based Linux d
 
 ### Basic hardware recommendations
 
+Once you have [sized your cluster](#sizing), derive the amount of RAM, storage capacity, and disk I/O from the number of vCPUs.
+
 This hardware guidance is meant to be platform agnostic and can apply to bare-metal, containerized, and orchestrated deployments. Also see our [cloud-specific](#cloud-specific-recommendations) recommendations.
 
 | Value | Recommendation | Reference
 |-------|----------------|----------
-| RAM per vCPU | 4 GiB | [CPU and memory](#cpu-and-memory)
+| RAM per vCPU | 4 GiB | [Memory](#memory)
 | Capacity per vCPU | 150 GiB | [Storage](#storage)
 | IOPS per vCPU | 500 | [Disk I/O](#disk-i-o)
 | MB/s per vCPU | 30 | [Disk I/O](#disk-i-o)
 
-Before deploying to production, test and tune your hardware setup for your application workload. For example, read-heavy and write-heavy workloads will place different emphases on [CPU](#cpu-and-memory), [RAM](#cpu-and-memory), [storage](#storage), [I/O](#disk-i-o), and [network](#networking) capacity.
+Before deploying to production, test and tune your hardware setup for your application workload. For example, read-heavy and write-heavy workloads will place different emphases on [CPU](#sizing), [RAM](#memory), [storage](#storage), [I/O](#disk-i-o), and [network](#networking) capacity.
 
-#### CPU and memory
+#### Memory
 
-Each node should have **at least 4 vCPUs**. For best performance, we recommend at least 8 vCPUs per node. Provision **4 GiB of RAM per vCPU**.
-
-- To optimize for throughput, use larger nodes with up to 32 vCPUs. To further increase throughput, add more nodes to the cluster instead of increasing node size.
-
-      {{site.data.alerts.callout_info}}
-      Based on internal testing, 32 vCPUs is a sweet spot for OLTP workloads. It is not a hard limit, especially for deployments using physical hardware rather than cloud instances.
-      {{site.data.alerts.end}}
-
-      {{site.data.alerts.callout_info}}
-      The benefits to having more RAM decrease as the number of vCPUs increases.
-      {{site.data.alerts.end}}
-
-- To optimize for resiliency, use many smaller nodes instead of fewer larger nodes. Recovery from a failed node is faster when data is spread across more nodes.
+Provision at least {% include {{ page.version.version }}/prod-deployment/provision-memory.md %}. The minimum acceptable ratio is 2 GiB of RAM per vCPU, which is only suitable for testing.
 
 - To optimize for the support of large numbers of tables, increase the amount of RAM. For more information, see [Quantity of tables and other schema objects](schema-design-overview.html#quantity-of-tables-and-other-schema-objects). Supporting a large number of rows is a matter of [Storage](#storage).
 
-- Avoid "burstable" or "shared-core" virtual machines that limit the load on CPU resources.
+    {{site.data.alerts.callout_info}} 
+    The benefits to having more RAM decrease as the [number of vCPUs](#sizing) increases.
+    {{site.data.alerts.end}}
```

Comment:
This note is indented the next note is not indented with the bullet point, is that purposeful?